nest/src/AggregateToken.sol

```diff
@@ -201,6 +201,10 @@ contract AggregateToken is ComponentToken, IAggregateToken, ERC1155Holder {
         IComponentToken componentToken,
         uint256 amount
     ) external nonReentrant onlyRole(ADMIN_ROLE) {
+        // Verify the componentToken is in componentTokenMap
+        if (!_getAggregateTokenStorage().componentTokenMap[componentToken]) {
```

Comment:
Maybe `_getAggregateTokenStorage().componentTokenMap[componentToken] != address(0)` is more explicit? Both are fine